rocky/rocky/auth/remote_user.py

```diff
@@ -0,0 +1,34 @@
+import logging
+
+from django.conf import settings
+from django.contrib.auth.backends import RemoteUserBackend as BaseRemoteUserBackend
+from django.contrib.auth.models import Group
+from tools.models import Organization, OrganizationMember
+
+logger = logging.getLogger(__name__)
+
+
+class RemoteUserBackend(BaseRemoteUserBackend):
+    """
+    Custom RemoteUserBackend that adds users to default organizations and groups.
+    """
+
+    def configure_user(self, request, user, created=True):
+        if not user.is_superuser and settings.REMOTE_USER_DEFAULT_ORGANIZATIONS:
+            user_orgs = [o.name for o in user.organizations]
```

Comment:
That would mean it isn't possible to add users to a new organisation later, which I think would be useful to have (e.g. on our testserver, which is the primary use case for this PR).

Reply:
I think the main thing that is being handled here is adding new organizations to `settings.REMOTE_USER_DEFAULT_ORGANIZATIONS`. Which makes sense I think since I already logged into our test environment but still want to be added to these organizations when I log in again.

Reply:
I think that if you configure that the users and organizations are externally defined then the expectation is that the external source defines this and not a mix of an external source and what is defined in the Rocky database. If you delete a user account this will also be recreated by the Django remote user backend. If you are going to mix this it going to be hairy how it should work when things change.

We do need to document that this is how it works, I have created #1180 for this.